v2/index.html

```diff
@@ -867,99 +872,116 @@ <h4>Rhythmic Values</h4>
+                            <td>double-sharp</td>
+                        </tr>
+                        <tr class="notation-example">
+                            <td class="notation-code">
+                                <script type="application/json">
+                                    {
+                                        "clef": "G-2",
+                                        "data": "bbB"
+                                    }
+                                </script>
+                                <code>bb</code>
+                            </td>
+                            <td class="notation-result"></td>
+                            <td>double-flat</td>
+                        </tr>
+                        <tr class="notation-example">
+                            <td class="notation-code">
+                                <script type="application/json">
+                                    {
+                                        "clef": "G-2",
+                                        "data": "nB"
+                                    }
+                                </script>
+                                <code>n</code>
+                            </td>
+                            <td class="notation-result"></td>
+                            <td>natural</td>
+                        </tr>
                     </tbody>
+
                 </table>
-            </aside>
+                <figcaption>Accidental values</figcaption>
+            </figure>
+            <p>
+                The sounding pitch of a note MAY be altered by both a key signature or an accidental. In the case of
+                a note being altered by both, the alteration of the accidental MUST be interpreted as an alteration
+                of the pitch in the key signature.
+            </p>
+            <p>
+                A note altered by an accidental SHOULD NOT be altered subsequent times within the same bar. The
+                alteration to the sounding pitch of the note is continued on subsequent notes with the same name
+                until the next <a>bar line</a>.
+            </p>
+            <p>
+                A flat accidental MUST NOT be used to lower the sounding pitch of a sharpened note by one semitone.
+                Likewise, a sharp accidental MUST NOT be used to raise the sounding pitch by one semitone. In both
+                of these cases a natural accidental MUST be used, even if this differs from the content of the
+                <a>original source</a>.
+            </p>
+            <p>
+                When an <a>original source</a> does not contain a written accidental, but one is deemed necessary
+                (for example, because of harmonic context or content that is not part of the incipit) the encoder MUST
+                supply the written accidental to ensure the melodic sequence accurately reflects the intended sequence
+                of intervals.
```

Comment:
Maybe the same here. This is more of a cataloguing guidelines, and not really the encoding specifications.

Reply:
I debated that same question, but decided to add it because I think it is necessary to specify that the melodic content of the incipit MUST reflect the melodic content of the original, whether or not the actual symbols exist in the original. 

I know it's skating close to the line, but this further clarifies that the content of the incipit should not reflect the visual appearance, but the intended melodic content.